examples/ALOS2437590500-220630_WWDR2.2GUA/ALOS2437590500-220630_WWDR2.2GUA.json

```diff
@@ -0,0 +1,182 @@
+{
+  "type": "Feature",
+  "stac_version": "1.0.0",
+  "id": "ALOS2437590500-220630_WWDR2.2GUA",
+  "properties": {
+    "proj:epsg": 32651,
+    "proj:bbox": [
+      374612.5,
+      2689112.5,
+      780462.5,
+      3087012.5
+    ],
```

Comment:
This is more personal preference than a rule, but I don't find `proj:bbox` very useful ... do you have a use-case for it?

Reply:
I was following https://github.com/stac-extensions/projection#best-practices , not sure if I've seen real world use of it, but reading that doc I can see some potential uses like building a VRT from the STAC records.

Reply:
Ah gotcha -- yeah I guess I was more used to seeing `transform` used instead of `bbox`, but it's no problem to include. Thanks!